bwmon/www/bwmon.js

```diff
@@ -698,7 +703,25 @@ bwmon.controller('MainController', ['$scope', '$interval', '$http', '$location',
 		$scope.$watch('displayRate',function() {
 			$scope.setCookie('bwmon-displayRate', $scope.displayRate, $scope.SECONDS_IN_MONTH);
 		});
+
+		var stylesheet = $scope.readCookie('bwmon-displayStyleSheet');
+		if (stylesheet)
+			$scope.displayStyleSheet = stylesheet;
+
+		$scope.$watch('displayStyleSheet', function() {
+			$scope.setCookie('bwmon-displayStyleSheet', $scope.displayStyleSheet, $scope.SECONDS_IN_MONTH);
+			changeCSS($scope.displayStyleSheet+'.css', 1);
+		});
 	};
 
 	$scope.init();
-}]);
\ No newline at end of file
+}]);// Changes the css file on the fly. Index must be 1 because 0 is bootstrap
+// Changes the css file on the fly. Index must be 1 because 0 is bootstrap
+function changeCSS(cssFile, cssLinkIndex) {
```

Comment:
Nitpicky code review point:

The rest of the file is using jsDoc commenting standard for functions the header will end up looking like:

```
/**
 * Changes the css file on the fly. Index must be 1 because 0 is bootstrap's css.
 * @param {string} cssFile url path for the css to load.
 * @param {number} cssLinkIndex index of the link to replace.
 */
```

In the event you're using an IDE jsDoc allows a lot of the IDE's to determine the types to offer more useful auto complete and do validation. (You can just use my replacement code comment as is if you wish)

Comment: 

Otherwise I think it is very clever what you've done I was debating on bringing in a css preprocessor like less or sass to achieve this same thing but it's yet another dependency which can get annoying on a hobby project. However your solution seems to bypass that requirement pretty nicely.

There's a small potential that this can get problematic once there is more than 2 css files (updates will need to consider the new indexes). I still think your solution is the best despite this.